invenio_vocabularies/contrib/names/config.py

```diff
@@ -28,12 +28,16 @@ class NamesSearchOptions(SearchOptions):
 
     suggest_parser_cls = SuggestQueryParser.factory(
         fields=[
-            "name^100",
-            "family_name^100",
+            "name.suggest^80",
+            "name^80",
+            "family_name^70",
+            "given_name.suggest^100",
             "given_name^100",
             "identifiers.identifier^20",
             "affiliations.name^10",
```

Comment:
- For `affiliations.name`, we should instead use `affiliations.name.suggest`
- nit: reorder the fields by "boosting weight"